src/clients/UBAClient/UBAClientWithRefresh.ts

```diff
@@ -88,6 +88,8 @@ export class UBAClientWithRefresh extends BaseAbstractClient {
   ) {
     super();
     this.logger = this.hubPoolClient.logger;
+    // In order to load the enabled chains, we need to make sure that the config store client is updated.
+    assert(this.hubPoolClient.configStoreClient.isUpdated, "ConfigStoreClient must be updated");
```

Comment:
Can we also make sure this is checked in every single downstream client of the ConfigStoreClient?
- HubPool
- SpokePool
- more?

Reply:
Absolutely - I'll circle back to this